src/typing/typeloadModule.ml

```diff
@@ -187,6 +187,7 @@ end
 	Build module structure : should be atomic - no type loading is possible
 *)
 let module_pass_1 ctx m tdecls loadp =
+	Typecore.check_module_path ctx m.m_path loadp;
```

Comment:
That's problematic for netlibs. How could this check be avoided for netlibs? For types/fields we can just ignore extern ones, but for modules..?

Reply:
`m.m_extra.m_kind <> MExtern`

Reply:
This doesn't seem to prevent `check_module_path` from executing =/

Still got the error for `AutoMapper.dll` with:
`if m.m_extra.m_kind <> MExtern then Typecore.check_module_path ctx m.m_path loadp;`

Reply:
Interesting, it sets the `m_kind` only after that pass. I wonder why...

Reply:
I guess the check can be moved to `type_module`.

Reply:
Yes seems good

Reply:
Yep works for me.